monitor/census.go

```diff
@@ -491,6 +505,13 @@ func InitCensus(nodeType NodeType, version string) {
 			TagKeys:     append([]tag.Key{census.kProfiles}, baseTagsWithManifestID...),
 			Aggregation: view.Count(),
 		},
+		{
+			Name:        "segment_transcoded_all_appeared_total_per_orchestrator",
+			Measure:     census.mSegmentTranscodedAllAppearedPerOrchestrator,
+			Description: "Per orchestrator SegmentTranscodedAllAppeared",
+			TagKeys:     append([]tag.Key{census.kOrchestratorURI, census.kProfiles}, baseTagsWithManifestID...),
```

Comment:
Since There will no longer be a `_per_orchestrator` metric and we shouldn't surpress the manifest id tagging, shouldn't I create `baseTagsWithManifestIDAndOrchestratorURI`  and have something like that?
```
{
		Name:        "segment_transcoded_all_appeared_total",
		Measure:     census.mSegmentTranscodedAllAppeared,
		Description: "SegmentTranscodedAllAppeared",
		TagKeys:     append([]tag.Key{census.kProfiles}, baseTagsWithManifestIDAndOrchestratorURI...),
		Aggregation: view.Count(),
}
```